### PR TITLE
Rewards coming soon and SharedIcon component

### PR DESCRIPTION
**Pull request by jagodarybacka**
[🎨  Figma link](https://www.figma.com/file/q8gfu9XTbYBF07KMnDCtsD/UI---Dark?node-id=7523%3A182054)

### What is new

"Swap rewards coming soon" notification was added to Swap Assets tab. Notification can be dismissed with close button.

"Swap rewards for community" card was prepared to be added to Swap Assets tab. It contains tooltip with "Details" button.

Screenshot 2022-03-15 at 12 49 28

### CODE REVIEW

**Comment by jagodarybacka** [comment]
1. Rewards coming soon notification can be closed - how do we want it to behave? Should it be closed for good for given user? Just for the session? Do you have anywhere else in the code that kind of behaviour?
2. Rewards coming card/section - from our morning conversation with Vlad I know that it will be needed later. So we probably don't want to move transaction settings right now?
3. Tooltip is not perfect but it's improvements are still on the [approve-spend-limit-input-improve](https://github.com/tallycash/extension/tree/approve-spend-limit-input-improve) branch
4. Please take a look at the first version of SharedIcon component 😄

**Comment by henryboldi** [comment]
🤩 Is this ready for review or a draft?
<br/>

Btw, I think the `SharedIcon` component should be introduced in a separate PR, because it deserves its own discussion.

**Comment by jagodarybacka** [comment]
@henry, well it was blocked yesterday by linked branch, but it is merged now so it will be ready for review soon. But questions 1. and 2. are valid right now as I'm missing some information to finish implementation.

**Comment by VladUXUI** [comment]
I'll try and answer :)

1 - Should be closed for good. As far as i remember, we don't have this anywhere else in the code
2 - That is correct, let's not move transaction setting yet. I want to give this a bit more thought before doing it.

File: ui/pages/Swap.tsx [hunk 1]
```diff
@@ -478,10 +481,11 @@ export default function Swap(): ReactElement {
               />
             </div>
             <div className="settings_wrap">
-              <SwapTransactionSettingsChooser
+              <SwapRewardsCard />
```
Comment (jagodarybacka) [inline ui/pages/Swap.tsx:484]: ❗ `SwapRewardsCard` needs to be removed and Settings needs to be back here before merging this PR.

**Pull request closed without merging**